myapp/app/controllers/tasks_controller.rb

```diff
@@ -1,8 +1,17 @@
 class TasksController < ApplicationController
   def index
-    sort_by = params[:sort_by] || 'created_at'
-    order = (params[:order] || 'desc').downcase
-    @tasks = Task.order("#{sort_by} #{order}")
+    if !params.except(:controller, :action).empty?
```

Comment:
[NITS]
```suggestion
    if params.except(:controller, :action).present?
```

you can use `present?` method!